Datasets/data-connect-dataset-onedrivesyncerrors.md

```diff
@@ -13,6 +13,13 @@ ms.custom: datasets:dataset-name
 
 The OneDrive Sync Errors dataset includes information about errors on devices running OneDrive for Business. This includes one object for every type of error in a specific device. Data includes error codes, error messages and number of times an error was seen on the device. 
 
+NOTE:
+
+- Currently, the MGDC platform ONLY supports extracting valid users. Trying to extract invalid users will result in no data to be returned for such users. Valid users must have the following:
```

Comment:
Please remove changes from this file as it is a tenant shard dataset

Reply:
updated